controller/tests/utils.py

```diff
@@ -28,6 +29,15 @@ def get_mock_mediator_response_snapshot(capacity, name, wwn, volume_name, array_
     return snapshot
 
 
+def get_mock_volume_capability_object(mode=csi_pb2.VolumeCapability.AccessMode.SINGLE_NODE_WRITER, fs_type="ext4",
```

Comment:
no need for `object`, and I would rearrange the order to `volume_capability_mock`, since it's more of a mock than a real volume capability.

Reply:
done

Reply:
as I wrote, I think it's more accurate for the name to **end** with `mock`, since it's a mock of volume capability (and not a volume capability of mock)